### DISCUSSION TITLE
Camera Scanner for Asset Details

### DISCUSSION THREAD

**Discussion by Crazyplayer364** [snipe/snipe-it#15974] [Ideas]
Hello, 

Can I request a feature for camera support so you can scan in serial numbers, Model Numbers and other details on a device. 

As a small company we use phones to add asset details and being able to scan then with the phones will be very beneficial

**Comment by gajotnt** [comment]
I thing you can use the android app for that.
I use it to check in/out assets.

https://play.google.com/store/apps/details?id=com.diegogarciadev.assetsmanager.snipeit&pli=1

**Reply by Crazyplayer364** [reply]
We have tried to use the app before but have found that its very glitchy and doesn't seem to always work that is why we would like it within snipe
